fields/types/textarray/TextArrayType.js

```diff
@@ -25,16 +25,50 @@ function textarray(list, path, options) {
 util.inherits(textarray, super_);
 
 /**
+ * Validates that a value for this field has been provided in a data object
+ *
+ * @api public
+ */
+
+textarray.prototype.validateInput = function(data, required, item) {
+	var value = this.getValueFromData(data);
+
+	if (required) {
+		if (value === undefined && item && item.get(this.path)) {
```

Comment:
This will return a false positive because `item.get(this.path)` should always return an array even when empty, which would pass the truthy check. Need to validate length of the existing array.

this should fix it: `value === undefined && item && item.get(this.path) && item.get(this.path).length`